ckanext/canada/templates/package/resource_read.html

```diff
@@ -31,6 +31,11 @@
   {% endif %}
 {% endblock %}
 
+{% block breadcrumb_content %}
+  {{ super.super() }}
+  <li class="active"><a href="" title="{{ h.resource_display_name(res) }}">{{ h.resource_display_name(res)|truncate(80) }}</a></li>
```

Comment:
AFAIK `<a>` tags without an href are bad for a11y. Are we allowed to add a title to another type of tag?

Reply:
I think you can do javascript void into the href to fix the a11y complaints.

Reply:
Also put `tabindex="-1"` to prevent tabbing